[2024-07-08T09:02:23.133+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-07-08T09:02:23.153+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: DAG_8.print_hello manual__2024-07-08T09:02:20.800643+00:00 [queued]>
[2024-07-08T09:02:23.160+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: DAG_8.print_hello manual__2024-07-08T09:02:20.800643+00:00 [queued]>
[2024-07-08T09:02:23.161+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-07-08T09:02:23.175+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): print_hello> on 2024-07-08 09:02:20.800643+00:00
[2024-07-08T09:02:23.184+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'DAG_8', 'print_hello', 'manual__2024-07-08T09:02:20.800643+00:00', '--job-id', '29', '--raw', '--subdir', 'DAGS_FOLDER/dag8.py', '--cfg-path', '/tmp/tmp1ijtofhv']
[2024-07-08T09:02:23.186+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=844) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-07-08T09:02:23.186+0000] {standard_task_runner.py:91} INFO - Job 29: Subtask print_hello
[2024-07-08T09:02:23.187+0000] {standard_task_runner.py:63} INFO - Started process 846 to run task
[2024-07-08T09:02:23.226+0000] {task_command.py:426} INFO - Running <TaskInstance: DAG_8.print_hello manual__2024-07-08T09:02:20.800643+00:00 [running]> on host 0ffcb94bda06
[2024-07-08T09:02:23.307+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='DAG_8' AIRFLOW_CTX_TASK_ID='print_hello' AIRFLOW_CTX_EXECUTION_DATE='2024-07-08T09:02:20.800643+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-07-08T09:02:20.800643+00:00'
[2024-07-08T09:02:23.308+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-07-08T09:02:23.309+0000] {logging_mixin.py:188} INFO - Hello, World! Its the 8th DAG
[2024-07-08T09:02:23.309+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-07-08T09:02:23.310+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-07-08T09:02:23.317+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=DAG_8, task_id=print_hello, run_id=manual__2024-07-08T09:02:20.800643+00:00, execution_date=20240708T090220, start_date=20240708T090223, end_date=20240708T090223
[2024-07-08T09:02:23.362+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-07-08T09:02:23.377+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-07-08T09:02:23.379+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
